server/cli/src/main/java/org/elasticsearch/cli/MultiCommand.java

```diff
@@ -74,4 +76,48 @@ protected void execute(Terminal terminal, OptionSet options) throws Exception {
         }
         subcommand.mainWithoutErrorHandling(Arrays.copyOfRange(args, 1, args.length), terminal);
     }
+
+    @Override
+    public void close() throws IOException, RuntimeException {
```

Comment:
I don't think we should copy this here. I think if we are going to copy it, it should be available everywhere and replace the usage of the Lucene `IOUtils`. I opened #29012.